src/jobs/publish_s3.yml

```diff
@@ -0,0 +1,35 @@
+description: Publish files to S3
+
+executor: aws
+
+parameters:
+  public:
+    type: boolean
+    description: Whether to store in the public bucket
+  path:
+    type: string
+    description: Path to store the files in S3
+  files:
```

Comment:
files being of type string, and then separated by coma makes me really nervous. Can't we pass an array? or multiple "file" params? Or even just have this receiving a single file, and on the cases where we want multiple, we just call this multiple times?

Reply:
No, we can't pass an array unfortunately.
Calling the orb multiple times is wasteful because it will spin up a machine for every file.
Files are separated by spaces, like [here](https://github.com/codacy/codacy-coverage-reporter/pull/295/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R262)

Reply:
alright, if we are fine with this, let's do it! :D